content/source/docs/cloud/api/policies.html.md

```diff
@@ -22,6 +22,8 @@ page_title: "Policies - API Docs - Terraform Cloud"
 
 # Policies API
 
+-> **Note:** Sentinel is a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
 [Sentinel Policy as Code](../sentinel/index.html) is an embedded policy as code framework integrated with Terraform Cloud.
```

Comment:
> integrated with Terraform
s/ with / within?

Reply:
Not quite the right usage; also that text is outside the effect of the PR, so I'm gonna table that for now.